crates/transaction-pool/src/validate/eth.rs

```diff
@@ -752,6 +772,7 @@ impl<Client> EthTransactionValidatorBuilder<Client> {
             shanghai: AtomicBool::new(shanghai),
             cancun: AtomicBool::new(cancun),
             prague: AtomicBool::new(prague),
+            max_blob_count: AtomicU64::new(MAX_BLOBS_PER_BLOCK as u64),
```

Comment:
oh here we actually need to check if prague then BlobParams::prague, otherwise BlobParams::cancun blobcount